src/kde/kde.hh

```diff
@@ -0,0 +1,87 @@
+//--------------------------------------------*-C++-*---------------------------------------------//
+/*!
+ * \file   kde/kde.hh
+ * \author Mathew Cleveland
+ * \brief  Define class kernel density estimator class
+ * \note   Copyright (C) 2018-2020 Triad National Security, LLC.
+ *         All rights reserved. */
+//------------------------------------------------------------------------------------------------//
+
+// clang-format off
+
+#ifndef kde_kde_hh
+#define kde_kde_hh
+
+#include <vector>
+#include <array>
+#include "c4/global.hh"
+
+namespace kde {
+
+//================================================================================================//
+/*!
+ * \class kde - kernel density estimator class for generated smoothed
+ * reconstructions of point wise PDF data
+ * \brief
+ *
+ * Returns a KDE reconstruction of a multidimensional distribution
+ * 
+ */
+//================================================================================================//
+
+  enum kde_coordinates{CART, CYL, SPH};
```

Comment:
Consider using a scoped enum. (clang-tidy is going to ask for this anyway).

Reply:
Should I change this?